content/news/playtest-20151114.md

```diff
@@ -0,0 +1,49 @@
+We recieved some great feedback on our [Halloween playtest](/news/playtest-20151031/), and have been hard at work over the last two weeks fixing the issues that have been reported.  Thanks to everyone who has tried the playtest and reported bugs or other feedback!
+
+We are now happy enough with the stability and integration of [all the new
+features](/news/playtest-20151031/) to mark today's new playtest build as the first release candidate for our upcoming December release.  We encourage you to give this release a good try out, especially if you like to play competitively online, because if we don't hear your balance complaints now then you will be stuck with them for the next few months until our next release!  You can report any issues you have by [filing a bug](http://bugs.openra.net) or [joining our IRC channel](/community/).
+
+This release fixes several general issues, including:
+
+* Fixed a hang when exiting the game under Windows
+* Fixed an exploit that allowed the color validator to be bypassed
+* Fixed Battlefield News never updating for some players
+* Improved polish and integration of the global chat channel
+* Surprise bonus feature: Copy/Paste in the map editor!
+
+Fixes for the Dune 2000 mod include:
+
+* Fixed repair depots not repairing units, and buffed repair rate
+* Fixed missing unit crates
+* Fixed incorrect unit armor / damage statistics
+* Improved behaviour of Spice Blooms
+* Improved behaviour of Thumpers and Worms
+* Tweaked support power timers and Ornithoper Strike damage
+
+We further iterated the Red Alert mod balance:
+
+* Fixed Artillery and V2 not targeting units under the fog with enabled GPS
```

Comment:
AFAIK all other units have an autotarget range equal or smaller than their sight range, so there is no difference for them.